### PR TITLE
PIN-3477 Adjusted Save info on DynamoDB for the privacy policies

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

**Comment by beetlecrunch** [comment]
Please change the PR title (**PIN-XXXX - THE TITLE IN ENGLISH**).
🙏

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala [hunk 1]
```diff
@@ -84,11 +85,11 @@ final case class PrivacyNoticesApiServiceImpl(
...
         PersistentModel.UserPrivacyNotice(
-          pk = s"${PersistentModel.UserPrivacyNotice.pkPrefix}$pnUuid",
-          sk = s"${PersistentModel.UserPrivacyNotice.skPrefix}$userUuid#${latest.privacyNoticeVersion.version}",
-          pnId = pnUuid,
+          pnIdWithUserId = s"$pnUuid#$userUuid",
+          versionNumber = latest.privacyNoticeVersion.version,
+          privacyNoticeId = pnUuid,
           userId = userUuid,
-          acceptedAt = OffsetDateTimeSupplier.get(),
+          acceptedAt = OffsetDateTimeSupplier.get().truncatedTo(ChronoUnit.SECONDS),
```
Comment (beetlecrunch) [inline src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala]: [same](https://github.com/pagopa/interop-be-jobs/pull/114#discussion_r1213064052)

**Review by beetlecrunch — APPROVED** [review]

**Review by TonioGela — APPROVED** [review]

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala (hunk 1 above)
Comment (nttdata-rtorsoli) [reply src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala]: The attribute acceptedAt is the time of the operation and I supposed that up to the seconds was enough. If you want I remove it

Comment (nttdata-rtorsoli) [reply src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala:92]: The attribute acceptedAt is the time of the operation and I supposed that up to the seconds was enough. If you want I remove it

Comment (nttdata-rtorsoli) [reply src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala]: Fixed

**Review by galales — APPROVED** [review]

**Pull request merged**